src/adapter/input/post_router.py

```
@@ -0,0 +1,35 @@
+                500: {"description": "서버 내부 오류"}
+            })
+def create_post(post: Post, service: PostApplicationService = Depends(get_post_application_service)):
+    try:
+        return service.create_post(post)
+    except HTTPException as e:
+        raise e
+    except Exception as e:
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=str(e)
+        )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)